OlinkAnalyze/NEWS.md

```diff
@@ -1,3 +1,19 @@
+# Olink Analyze 4.0.0
+## Major Changes
+* olink_normalization can now be used for bridging Explore 3072 data to Explore HT data (#453, #452, #439, #449, #447,#446, #441, #440, #434, #422, #436, #435 @klevdiamanti, @kathy-nevola, @kristynchin-olink, @dtopouza, @amrita-kar)
```

Comment:
We should add Masoumeh here, maybe in some other points too if she worked on optimization and the tutorial.